runtime/slice.go

```diff
@@ -156,6 +156,58 @@ func initSliceType(map[string]*Object) {
 	SliceType.slots.Repr = &unaryOpSlot{sliceRepr}
 }
 
+func sliceIndex(f *Frame, o *Object) (i int64, raised *BaseException) {
```

Comment:
Looks like 9223372036854775808L  not in int range, 
sys.maxsize + 1 pretty much corner case with no actual use case.
Do we support it? If so, I need to add more checks with big.Int.